sklearn/neighbors/tests/test_dist_metrics.py

```diff
@@ -47,14 +53,16 @@ def dist_func(x1, x2, p):
                           'canberra': {},
                           'braycurtis': {}}
 
-
 @pytest.mark.parametrize('metric', METRICS_DEFAULT_PARAMS)
-def test_cdist(metric):
+@pytest.mark.parametrize('X1, X2', [(X1, X2), (X1_mmap, X2_mmap)])
+def test_cdist(metric, X1, X2):
     argdict = METRICS_DEFAULT_PARAMS[metric]
     keys = argdict.keys()
     for vals in itertools.product(*argdict.values()):
         kwargs = dict(zip(keys, vals))
-        if metric == "wminkowski":
+        if metric == "mahalanobis":
+            pytest.xfail("cdist with 'mahalanobis' fails on memmap data")
```

Comment:
Let's open an issue specifically for cdist/pdist + `'mahalanobis'` and link it here with a comment so we can track the issue.

Reply:
I recently have opened https://github.com/scipy/scipy/issues/13861.

I will add a comment here, mentioning it.